node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,8 +137,14 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType? widgetData.previews.targetType.split(';')[0]: false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes[type];
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
+            if (embeddable && supported) {
```

Comment:
I still encounter the issue where the page is trying to download the file when using a link like `http://www.colorado.edu/conflict/peace/download/peace_example.ZIP`. We should make sure that links like that also don't try to download the file on every page load.